packages/manager/core/app/src/api/service.ts

```diff
@@ -153,32 +154,32 @@ export type GenericProductDefinition = {
 };
 
 export async function getServiceDetails(serviceId: number): Promise<Service> {
-  const response = await fetch(`/engine/apiv6/services/${serviceId}`);
-  return response.json();
+  const { data } = await apiClient.v6.get(`/services/${serviceId}`);
```

Comment:
I'm wondering as we are always repeating the same pattern, does it make sense for apiClient to directly return data instead of an object ?

Reply:
Good point. Need to be discussed :-)
Cc @frenautvh, @cbourgois